### PR TITLE
add support@vumi.org to low credit notifications

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@rudigiesler @hodgestar ready for review

File: go/billing/tasks.py [hunk 1]
```diff
@@ -547,7 +548,9 @@ def create_low_credit_notification(
             'reference': notification.id,
         })
 
-    email = EmailMessage(subject, message, email_from, [email_to])
+    # remove duplicate emails from the recipient list
+    recipient_emails = list(set([email_to, support_email]))
+    email = EmailMessage(subject, message, email_from, recipient_emails)
```
Comment (rudigiesler) [inline go/billing/tasks.py]: As discussed on slack, we probably want the `support_email` to be a `bcc`. Should be as simple as adding `bcc=support_email` to the list of args, and fixing the relevant test.

Comment (KaitCrawford) [inline go/billing/tasks.py]: done :)
The test doesn't need to be fixed as it checks recipients():
"recipients() returns a list of all the recipients of the message, whether they’re recorded in the to, cc or bcc attributes"

**Comment by hodgestar** [comment]
Other than the bcc thing @rudigiesler spotted, looks good to me too.

**Comment by rudigiesler** [comment]
:+1: when travis is happy.

File: go/billing/tests/test_tasks.py [hunk 2]
```diff
@@ -1286,7 +1287,8 @@ def test_credit_cutoff_email_sent(self):
         self.assertEqual(len(mail.outbox), 1)
         [email] = mail.outbox
 
-        self.assertEqual(email.recipients(), [self.django_user.email])
+        self.assertEqual(email.recipients(),
+                         [self.django_user.email, 'support@vumi.org'])
```
Comment (hodgestar) [inline go/billing/tests/test_tasks.py:1292]: `email.bcc` contains the list of bcc'ed addresses so we could add an assert like:

``` python
self.assertEqual(email.bcc, ['support@vumi.org'])
```

in addition to the current assert.

**Comment by KaitCrawford** [comment]
@hodgestar are you happy for me to merge?

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**